_posts/2016-06-20-recent-graduate-at-skybet.md

```diff
@@ -0,0 +1,56 @@
+---
+layout:     post
+title:      A Recent Graduates Guide to Sky Betting And Gaming…
+date:       2016-06-20 10:00
+summary:    A recent graduates blog on their first few months at Sky Betting and Gaming as well as what to expect.
+category:   Graduates, Data Engineering
```

Comment:
Right now only one category is supported and if it is a brand new category it will need creating. Run `bundle exec rake createCategory["CATEGORY NAME"]`, or let me know and I can stick a PR into your branch.